typings/react-popper.d.ts

```diff
@@ -2,32 +2,36 @@ import * as React from "react";
 import * as PopperJS from "popper.js";
 
 interface ManagerProps {
-  children: React.ReactNode;
+    children: React.ReactNode;
 }
 export class Manager extends React.Component<ManagerProps, {}> { }
 
+interface RefProps {
+    ref: (ref: HTMLElement | null) => void;
+}
+
 interface ReferenceProps {
-  children: (props: ({
-    ref: (ref: HTMLElement | null) => void,
-  })) => React.ReactNode;
+    children: (props: RefProps) => React.ReactNode;
 }
 export class Reference extends React.Component<ReferenceProps, {}> { }
 
+interface ArrowProps extends RefProps {
```

Comment:
`PopperArrowProps` instead? So we keep all the Popper related types under the same namespace?